src/checkfiles/file.py

```diff
@@ -123,3 +123,11 @@ def make_payload(self):
         if self.file_not_found:
             payload.update({'upload_status': 'file not found'})
         return json.dumps(payload)
+
+    def get_upload_status(self):
```

Comment:
To be consistent with other things such as  content_md5sum and original_etag, this should also be a @property